modules/administration/pages/custom-ssl.adoc

```diff
@@ -1,97 +1,195 @@
 [[custom-ssl]]
-= Using a Custom SSL Certificate
+= Custom SSL Certificates
 
+You can use custom certificates with {productname} and {productname} Proxy.
 
+This section covers how to use a third party SSL certificate authority with a new {productname} installation, and replacing existing certificate with new custom certificates.
 
+Before you begin, ensure you have:
 
+* A certificate authority (CA) SSL public certificate
+* An SSL private key
+* An SSL public certificate
 
+Your key and certificate files must be in PEM format.
 
-The following section will guide you through using a custom certificate with {productname} {productnumber} and {susemgrproxy} {productnumber}.
+The host name of the SSL keys and certificates must match the host name of the machine you deploy them on.
```

Comment:
New RFCs more or less requires that the host name is configured in the "X509v3 Subject Alternative Name" Section of the Certificate. This is very technical but one of the typical problems of self generated Certificates.

In addition we could say that in that section you can also list multiple hostnames if the server has them.

Reply:
Like below, we should say that hostnames in the certificates must be FQDNs